pulumi/grapl/Pulumi.local-grapl.yaml

```diff
@@ -25,3 +25,13 @@ config:
   kafka:bootstrapServers:
     - "kafka-broker:9092"
   kafka:tlsEnabled: false
+
+  grapl:artifacts:
+    # You probably don't want any service versions in here.
+
+    # You DO need to specify AMIs (they're require_artifact()'d)
+    # Since this is local-grapl, we don't need real AMIs for Localstack EC2
+    grapl-nomad-consul-client:
+      us-east-1: ami-000fake
```

Comment:
What does Localstack actually do with these, then?

Reply:
Not a clue, probably swallows them up. I can't imagine Localstack spins up VMs or anything for EC2. 
Ultimately, this is here just so that we can get past the `require_artifact()` check.

Reply:
I guess my point is that if Localstack isn't going to do anything meaningful with this, should we put this behind an `if not LOCAL_GRAPL` check?

Reply:
Personally, I think that if localstack has nominal support for ec2 create-instance (even if it ignores the ami field and creates a docker instance or something), then the less stuff we can gate behind `if local_grapl` the better. 

But also - this is a battle I don't really care about, so - can do

Reply:
How would it know what docker container to use?

Reply:
update, I *dont* see anything about creating instances in ec2 localstack
https://github.com/localstack/localstack/blob/master/localstack/services/ec2/ec2_listener.py
so it might just be silently slurping up a request it doesn't recognize - so - feel better gating it